tests/cypress/plugins/imageGenerator/addPlugin.js

```diff
@@ -19,13 +19,15 @@ function imageGenerator(args) {
     const posY = args.posY
     const message = args.message
     const file = path.join(directory, fileName)
-    const image = new jimp(width, height, color, function (err, image) {
-        if (err) throw err
-        jimp.loadFont(jimp.FONT_SANS_64_BLACK, function (err, font) {
+    return new Promise((resolve) => {
+        const image = new jimp(width, height, color, function (err, image) {
             if (err) throw err
-            image.print(font, Number(posX), Number(posY), message)
-            .write(file)
+            jimp.loadFont(jimp.FONT_SANS_64_BLACK, function (err, font) {
+                if (err) throw err
```

Comment:
I suppose ``reject(err)`` would be better (where reject is the second argument of the promise)?

Reply:
It's better this way. The function is asynchronous. Thanks